app/src/main/java/com/siri/RearrangeScreens/ThirdActivity.kt

```diff
@@ -0,0 +1,19 @@
+package com.siri.RearrangeScreens
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+
+class ThirdActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_three)
+        val button: Button = findViewById(R.id.simpleButton)
+        button.setOnClickListener {
+            val i = Intent(this, FourthActivity::class.java)
+            startActivity(i)
+        }
```

Comment:
Nitpick.

This will probably depend on your linter, but I personally find code easier to read when it's spaced out like this:

```
override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    setContentView(R.layout.activity_three)
    val button: Button = findViewById(R.id.simpleButton)

    button.setOnClickListener {
        val i = Intent(this, FourthActivity::class.java)
        startActivity(i)
    }
}
```
That way it's not just a wall of text and I can see clearly and quickly that (a) there is a call to your super class first and that (b) you are adding a closure as an event listener to your button.

Reply:
I can format the same way  entire project once i use the linter